call-home/Cargo.toml

```
@@ -19,9 +19,9 @@ path = "src/bin/stats/main.rs"
 
 [dependencies]
 constants = { path = "../constants" }
-openapi = {path = "../dependencies/control-plane/openapi"}
-kube = { version = "0.85.0", features = ["runtime", "derive"] }
-k8s-openapi = { version = "0.19.0", features = ["v1_20"] }
+openapi = { path = "../dependencies/control-plane/openapi" }
+kube = { version = "0.87.0", features = ["runtime", "derive"] }
```

I used

```suggestion
kube = { version = "0.87.2", features = ["runtime", "derive"] }
```

in the other PR but :shrug:

---

hmm I see 0.87.0 in the toml?
https://github.com/openebs/mayastor-control-plane/commit/14066087227b5020c06159e7594448082f7d87ce
and 
```
❯ ag "kube ="                                                                                                                                                                                                                            ─╯
utils/platform/Cargo.toml
13:kube = { version = "0.87.0", features = ["derive"] }

k8s/forward/Cargo.toml
10:kube = { version = "0.87.0", features = [ "ws" ] }

k8s/operators/Cargo.toml
31:kube = { version = "0.87.0", features = ["derive", "runtime"] }

control-plane/csi-driver/Cargo.toml
51:kube = { version = "0.87.0", features = ["runtime", "derive"] }
```

---

Anyway we end up with 0.87.2 because when updating the lock cargo pulls the latest compatible, and 0.87.2 is indeed compatible with 0.87.0

---

oh you're right, it is 0.87.0. must have lost that at some point.